src/pages/guides/review/index.md

```diff
@@ -35,28 +57,84 @@ By creating a project using the SDK in Adobe Developer Console, you are agreeing
 
 Additionally, your integration must not violate the trademarks and copyrights of Adobe, Inc., or those of any other party. Please adhere to all trademark guidelines as noted here: <https://www.adobe.com/legal/permissions/trademarks.html>
 
-## Submit your app
+## Submission Guidelines
+
+To make sure your review process goes smoothly, check off the tasks in this list before submitting.
+
+If you have any questions, feel free to contact us at [ccintrev@adobe.com](mailto:ccintrev@adobe.com) or reach out on our [Adobe Express Embed community forum](https://community.adobe.com/t5/express-embed-sdk/ct-p/ct-express-embed-sdk?page=1&sort=latest_replies&lang=all&tabid=all). 
+
+<InlineAlert variant="help" slots="text" />
+
+- Confirm that the `appName` used to initialize the SDK in your application code reflects your app's name. 
+- Note any projects created through your integration will be saved in the user's Express account, inside an Express folder with this name.
+- Confirm that the SDK is never initialized more than once per page load.
+
+### 1. Submit for approval
+
+Navigate to your project in Adobe Developer Console -- select **Submit for review** from the __Project overview__ tab or select the __Approval__ tab on the left panel.
```

Comment:
Nit: is the `--` intended as an em dash? Maybe:  
... in Adobe Developer Console; select...
... in Adobe Developer Console—select...
(totally marginal comment)